File: src/server/DeepSeatServer/src/main/kotlin/com/deepseat/server/DeepSeatServer/vo/Comment.kt
package com.deepseat.server.DeepSeatServer.vo

class Comment(commentID: Int, userID: String, docID: Int, content: String, wrote: String, edited: Boolean) {
    var commentID: Int
    var userID: String
    var docID: Int
    var content: String
    var wrote: String
    var edited: Boolean

    init {
        this.commentID = commentID
        this.userID = userID
        this.docID = docID
        this.content = content
        this.wrote = wrote
        this.edited = edited
    }
}

File: src/app/iOS/DeepSeat/DeepSeat/Model/Comment.swift
//
//  Comment.swift
//  DeepSeat
//
//  Created by Jisoo Woo on 2022/05/20.
//

import Foundation

class Comment{
    var commentID: Int
    var userID: String
    var docID: Int
    var content: String
    var wrote: String
    var edited: Bool
    
    init(commentID: Int, userID: String, docID: Int, content: String, wrote:String, edited: Bool){
        self.commentID = commentID
        self.userID = userID
        self.docID = docID
        self.content = content
        self.wrote = wrote
        self.edited = edited
    }
}
